denonavr/tonecontrol.py

```diff
@@ -153,6 +153,30 @@ async def async_set_bass(self, value: int) -> None:
         await self.async_enable_tone_control()
         await self.async_set_tone_control_command("bassvalue", value)
 
+    async def async_bass_up(self) -> None:
+        """
+        Increase level of Bass.
+
+        Note:
+        Doesn't work, if Dynamic Equalizer is active.
+        """
+        if self.bass == 12:
+            return
+        await self.async_enable_tone_control()
+        await self.async_set_tone_control_command("bassvalue", self.bass + 1)
+
```

Comment:
How about an `await self.async_update()` at this place?
Otherwise the command will work only once.